### ISSUE TITLE
Direct Label Editing does not work on mobile

### ISSUE THREAD

**Issue by nikku** [bpmn-io/bpmn-js#618]

**Comment by alexandrMykhailov** [comment]
Hello!
Please, fix label editing on mobile (we use iPad), because we are really stuck with it.

**Comment by seibernaut** [comment]
This is a show stopper. Symbols without labels will not make A BPMN diagram work.

**Comment by nikku** [comment]
We welcome contributions that improve the touch support.

**Comment by MaxTru** [comment]
Also reported by a customer. Use case:
> Usage of large monitors to look at BPMN diagrams. Double-click not working to edit labels of elements.

**Comment by MaxTru** [comment]
I had a first look:

* We ignore `element.click` events fired by the touchModule, because they do not contain a button type (e.g., `0` for leftMouseClick or `2` for rightMouseClick) => https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/interaction-events/InteractionEvents.js#L73. So here we could in principle monkeyPatch the button time when [firing from touchEvents](https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/touch/TouchInteractionEvents.js#L173), or alternatively adjust the ignore function, to also allow events with source === touch.
* However, also a `tap` fires not only a `element.click` but also a `element.mousemove` which does not seem right

Long story short: I think we fundamentally need to work on how we support touch, not a quickFix.

**Comment by EricKalesky** [comment]
Without the double click on touch screens we cannot really use Cawemo during workshops. Currently we use whiteboards, pen and paper. But we have three 75'' touch screens next to each other. If we could use Cawemo already during the first process capture it would save a lot of time of workshop postprocessing. Looking foreward to the #springcleaning

**Comment by MaxTru** [comment]
Fixed upstream via https://github.com/bpmn-io/diagram-js/pull/606
